ingestion/examples/airflow/dags/airflow_lineage_operator.py

```diff
@@ -0,0 +1,118 @@
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+You can run this DAG from the default OM installation
+"""
+
+from datetime import datetime, timedelta
+from textwrap import dedent
+
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+
+# Operators; we need this to operate!
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+
+# These args will get passed on to each operator
+# You can override them on a per-task basis during operator initialization
+from airflow_provider_openmetadata.lineage.operator import OpenMetadataLineageOperator
+from metadata.generated.schema.entity.services.connections.metadata.openMetadataConnection import (
+    OpenMetadataConnection,
+)
+from metadata.generated.schema.security.client.openMetadataJWTClientConfig import (
+    OpenMetadataJWTClientConfig,
+)
+
+default_args = {
+    "retries": 1,
+    "retry_delay": timedelta(minutes=5),
+}
+
+
+def explode():
+    raise Exception("Oh no!")
+
+
+with DAG(
+    "lineage_tutorial_operator",
+    default_args=default_args,
+    description="A simple tutorial DAG",
+    schedule_interval=timedelta(days=1),
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example"],
+) as dag:
+
+    # t1, t2 and t3 are examples of tasks created by instantiating operators
+    t1 = BashOperator(
+        task_id="print_date",
+        bash_command="date",
+        outlets={"tables": ["sample_data.ecommerce_db.shopify.dim_address"]},
+    )
+
+    t2 = BashOperator(
+        task_id="sleep",
+        depends_on_past=False,
+        bash_command="sleep 1",
+        retries=3,
+        inlets={"tables": ["sample_data.ecommerce_db.shopify.dim_customer"]},
+    )
+
+    risen = PythonOperator(
+        task_id="explode",
+        provide_context=True,
+        python_callable=explode,
+        retries=0,
+    )
+
+    dag.doc_md = (
+        __doc__  # providing that you have a docstring at the beginning of the DAG
+    )
+    dag.doc_md = """
+    This is a documentation placed anywhere
+    """  # otherwise, type it like this
+    templated_command = dedent(
+        """
+    {% for i in range(5) %}
+        echo "{{ ds }}"
+        echo "{{ macros.ds_add(ds, 7)}}"
+        echo "{{ params.my_param }}"
+    {% endfor %}
+    """
+    )
+
+    t3 = BashOperator(
+        task_id="templated",
+        depends_on_past=False,
+        bash_command=templated_command,
+        params={"my_param": "Parameter I passed in"},
+    )
+
+    t1 >> [t2, t3]
+
+    server_config = OpenMetadataConnection(
```

Comment:
@pmbrull we should use the airflow connection for this. Otherwise each DAG needs to put this in code.

Reply:
Its time for us to make airflow connection another option for OM

Reply:
@harshach added the `OpenMetadataHook` to let users create OpenMetadata connections from the Airflow UI and dynamically load the `OpenMetadataConnection`. Updated the docs accordingly. Let me know what you think.
Thanks